### ISSUE TITLE
05 -- Jeremy Rist

### ISSUE THREAD

**Issue by jeremyrist** [TIY-Durham/2015-FALL-FEE#124]
- **Yak-Shaving**
  - Create a new _WIP Issue_ named `05 -- YOUR NAME` with:
    - [x] link to PR in `USERNAME.github.io` from `journal-week-1` into `master`
    - [x] link to PR in `StudentRoster` from `USERNAME:master` into `TIY-Durham:master`
    - [x] link to PR in `TIY-Assignments` from `05--lost-yak` into `master`
  -  Create the following _WIP Branch_ in:
    - `USERNAME.github.io` named `journal-week-1`
    - `TIY-Assignments` named `05--lost-yak`
  - Create or update the following _WIP Files_ in:
    - `USERNAME.github.io`:
      - `journal-week-1.md`
    - `TIY-Assignments`
      - `ReadingJavaScript/README.md`
      - `NestedAccordion/` from Blueprints
      - `ViewModeSwitch/` from Blueprints
      - `MultiColumnForm/` from Blueprints
      - `TodoMVC/` from Project Template
- **Journal, Week 1**
  - [x] created branch in `USERNAME.github.io` named `journal-week-1`
  - [x] created new file named `journal-week-1.md`
  - [x] commited, pushed, and opened PR to `master`
  - [ ] added draft journal entry
- **Reading JavaScript: DOM EDITION**
  - [x] the `Window` object
  - [ ] the `Location` object
  - [ ] the `Document` object
  - [ ] the `HTMLElement` type
  - [ ] the `HTMLCollection` and `NodeList` types
- **Making Blueprints**
  - [x] Created branch in `TIY-Assignments` named `05--lost-yak`
  - [x] Downloaded starter code (ZIP) for:
    - [x] Nested Accordion
    - [x] View Mode Switch
    - [x] Responsive Multi-Column Form
  - [x] Extracted starter code into folders:
    - [x] `NestedAccordion`
    - [x] `ViewModeSwitch`
    - [x] `MultiColumnForm`
  - [ ] **BEAST MODE** -- Working code as-is!
- **GitHub-ing to Know You**
  - [x] merged upstream additions from `TIY-Durham/Student-Roster:master`
  - [x] updated your classmate's Markdown file and opened a PR to merge
  - [ ] thumbs up (:+1:) on your PR from your "Special Someone"
- **Hello, TodoMVC!**
  - [x] Downloaded the [TodoMVC template](https://github.com/tastejs/todomvc-app-template) from GitHub (see instructions!)
  - [x] Unzipped into `TIY-Assignments` as `TodoMVC`
  - [x] Committed and pushed to _WIP Branch_
  - **BEAST MODE**
    - [ ] Followed the instructions in `README.md`
    - [ ] Well, it looks right... Does nothing!
- **API Gymnastics**
  - **Yak Shaving**
    - [x] Create your _WIP Branch_! Maybe even check it out?
    - [x] Create a folder named `APIGymnastics`
    - [x] Copy-pasta **Starting Point** in `APIGymnastics/Etsy.js`
    - [x] Hey, what about that data?
    - [x] Seems like a good time to commit and push...
    - [x] Y'know, why not open a PR, too?
  - **So many questions...**
    - [x] What is the total number of items?
    - [ ] What is the average price of all the items?
    - [ ] What items cost between $14 and $18 dollars? What are their names?
    - [ ] What is the name and price of the only item with a `GBP` currency code?
    - [ ] What are the names of all the items made of wood?
    - [ ] What are the names of the items that are made of 8 or more materials?
    - [ ] How many items were made by their sellers?

**Comment by jeremyrist** [comment]
https://github.com/jeremyrist/jeremyrist.github.io/pull/11

**Comment by jeremyrist** [comment]
https://github.com/jeremyrist/TIY-Assignments/pull/7

**Comment by jeremyrist** [comment]
https://github.com/jeremyrist/TIY-Assignments/pull/8

**Comment by jeremyrist** [comment]
https://github.com/TIY-Durham/StudentRoster/pull/27

**Comment by jeremyrist** [comment]
I have no idea what these DOM elements are and I'm not sure how those sample websites do what they are doing.

**Comment by jeremyrist** [comment]
When I was working on the API Gymnastics problem I got stuck trying to find the average of the prices. I was able to figure out that the items.json file was one gigantic array, which meant I should have been able to reference different elements of the array by calling them with [ ] or by using a method. I was able to figure out that the price was always listed as the 11th element of the array and so I tried to call it by using the .slice method but I wasn't able to write code that made it all happen.
